src/pybind/mgr/dashboard/frontend/cypress/plugins/index.js

```diff
@@ -14,6 +14,13 @@ module.exports = (on, _config) => {
       return launchOptions;
     }
   });
+
+  on('task', {
+    log({ message, optional }) {
+      optional ? console.log(message, optional) : console.log(message);
+      return null;
```

Comment:
What's this for?

Reply:
the grafana-e2e commands like `e2e.components.Panels.Panel.title...` were failing because it couldn't find a command called `log`. So I had to manually add it on our code. Then it started passing.